src/components/Counter.tsx

```diff
@@ -0,0 +1,20 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { AppDispatch, RootState } from '../state/store';
+import { decrement, incrementAsync } from '../state/counter/counterSlice';
+
+const Counter = () => {
+  const count = useSelector((state: RootState) => state.counter.value);
+  const dispatch = useDispatch<AppDispatch>();
+
+  return (
+    <div>
+      <h2>{count}</h2>
+      <div>
+        <button onClick={() => dispatch(incrementAsync(10))}>Increment</button>
+        <button onClick={() => dispatch(decrement())}>Decrement</button>
```

Comment:
inconsistency in naming, please make sure your naming is consitent, do not call one **incrementAsync** and the other **decrement**.

Reply:
done